### PR TITLE
xplat: Try CXX/CC in case Clang wasn't on the PATH

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -377,11 +384,19 @@ else
...
-            echo "You could use clang++ from a custom location."
-            PRINT_USAGE
-            exit 1
+            # try env CXX and CC
+            if [[ ! -f $CXX ]]; then
+                ERROR_CLANG
+            fi
+
+            _CXX=$CXX
+            _CC=$CC
```
Comment (digitalinfinity) [inline build.sh]: is it worth validating that both cxx and cc are set in this case?

Comment (obastemur) [inline build.sh]: We weren't validating _CC so far. At least I could validate the availability of the one from ENV. Let me update. Thanks!

**Comment by obastemur** [comment]
@digitalinfinity Thanks for the review

**Pull request merged**